messanger/js.js

```diff
@@ -1,15 +0,0 @@
-document.getElementById("submit").addEventListener("click", function messageW() {
-    let messageText = document.getElementById("link").value;
-    let window = document.getElementById("window");
-
-    window.insertAdjacentHTML("beforeend", `<div class="message"> ${messageText} </div>`);
```

Comment:
create separate function for creating DOM element `<div class="message"> ${messageText} </div>`
ex: `const getMessageComponent = (msg) => "<div class="message"> ${messageText} </div>";`